FogCreek/Glitch-Community
# Fallacious slice: Shuffled around analytics code so it doesn't block the initial render

## Links
https://fallacious-slice.glitch.me/
https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/119

## Changes:
- Updated the segment snippet to the latest version, and pushed the `analytics.load()` call to the bottom of the page. This ensures that the analytics script doesn't load before the client code, which forces it to wait until the analytics are fully parsed/initialized
- Moved the browser checking code after the client js, and combined it together so `window.bootstrap()` is not called if we're showing an error page or redirecting

## How To Test:
- Try loading the page and check that analytics are still going through (POST to api.segment.io)
- Use lighthouse and compare with community.glitch.me (avoid glitch.com for a more even test)
